src/fa-screenshot.sh

```diff
@@ -150,7 +150,7 @@ tell application theApp
 	activate
 	reopen
 	set xAxis to $xAxis
-	set yAxis to 360
+	set yAxis to 200
```

Comment:
Is this a universal change that applies to all screenshots?

Reply:
This adjusts how the script positions the safari window on your monitor. It doesn't affect the screenshot size.